ape_safe/accounts.py

```diff
@@ -240,6 +240,28 @@ def compute_prev_signer(self, signer: Union[str, AddressType, BaseAddress]) -> A
         # NOTE: SENTINEL_OWNERS is the "previous" address to index 0
         return AddressType("0x0000000000000000000000000000000000000001")  # type: ignore[arg-type]
 
+    def load_submitter(
+        self,
+        submitter: Union[AddressType, str, None] = None,
+    ) -> AccountAPI:
+        if submitter is None:
+            if len(self.local_signers) == 0:
+                raise NoLocalSigners()
```

Comment:
we should also check for delegates, because a delegate could submit a tx, even if it has zero signatures.

Reply:
I haven't tested this out, but if `submitter=` a delegate account or alias it should bypass this logic and submit it. I check elsewhere if they're a signer only for signature aggregation. But probably I didn't do the delegate bypass logic correctly

Reply:
Tracking here: #16